presto-docs/src/main/sphinx/release/release-0.229.rst

```diff
@@ -0,0 +1,34 @@
+=============
+Release 0.229
+=============
+
+General Changes
+_______________
+* Fix an issue that would cause query failure when calling :func:`geometry_to_bing_tile` on certain degenerate geometries.
+* Add geospatial function :func:`line_interpolate_point`.
+* Add support for ``CREATE FUNCTION``.
+* Add support for using configuration property ``stage.max-tasks-per-stage`` to limit the number of tasks for scan.
+* Add support for passing ``X_Forwarded_For`` header from Proxy to coordinator.
+* Add configuration to use common ``HTTPS`` settings for internal communications.
```

Comment:
@hellium01 What are the specific configuration properties that we added?

Reply:
Following configuration options are added:
```
internal-communication.https.trust-store-path
internal-communication.https.included-cipher
internal-communication.https.excluded-cipher
```